kernel/build/config-arm64

```diff
@@ -3070,7 +3071,7 @@ CONFIG_USB_SIERRA_NET=y
 CONFIG_USB_VL600=y
 CONFIG_USB_NET_CH9200=y
 # CONFIG_USB_NET_AQC111 is not set
-# CONFIG_USB_RTL8153_ECM is not set
+CONFIG_USB_RTL8153_ECM=y
```

Comment:
is this intentional?

Reply:
yes, `make kernel-olddefconfig` did that